Source/WebCore/animation/AnimationTimelinesController.cpp

```diff
@@ -355,35 +360,30 @@ void AnimationTimelinesController::registerNamedViewTimeline(const AtomString& n
     }
 }
 
-void AnimationTimelinesController::unregisterNamedViewTimelineForSubject(const AtomString& name, const Element& subject)
+void AnimationTimelinesController::unregisterNamedTimeline(const AtomString& name, const Element& element)
 {
-    auto it = m_nameToViewTimelinesMap.find(name);
-    if (it == m_nameToViewTimelinesMap.end())
+    auto it = m_nameToTimelineMap.find(name);
+    if (it == m_nameToTimelineMap.end())
         return;
-
-    auto& timelines = it->value;
```

Comment:
I would have kept this line and avoid using `it->value` twice. Makes it clearer what we're manipulating as well. It matches `timelineForName()` as well.

Reply:
Fixed.